src/__tests__/Flex.spec.js

```diff
@@ -140,4 +140,24 @@ describe('<Flex /> component', () => {
       expect(container).not.toHaveStyle('flex-direction: column'); // default/fallback
     });
   });
+
+  describe('gap prop', () => {
+    test('has gap prop set to 1 rem', () => {
+      const container = renderFlex({ gap: '1rem' });
+      expect(container).toHaveStyle('gap: 1rem');
+      expect(container).not.toHaveStyle('gap: 0rem');
+    });
+
+    test('does not has gap prop', () => {
+      const container = renderFlex();
+      expect(container).toHaveStyle('gap: 0');
+      expect(container).not.toHaveStyle('gap: 1rem');
+    });
+
+    test('bad value', () => {
+      const container = renderFlex({ gap: 'oops' });
+      expect(container).not.toHaveStyle('gap: 1rem');
+      expect(container).not.toHaveStyle('gap: 0');
```

Comment:
Great tests! 😄 

Perhaps we can also add an expected fallback/default value? (the initial value we discussed above).

Reply:
how can i test this? as far as i know, if i make the default value for the gap prop equal to undefined, the svelte style directive would just ignore it and not pass anything. 
should i `expect(container).not.toHaveStyle('gap: 0')`?

Reply:
Because the initial value for `gap` is `'normal normal'`, I think it would be a great idea to do a positive check for that value (I _think_ that `testing-library`'s `toHaveStyle` should return true because it is using [getComputedStyle](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) under the hood, which always returns whatever the initial value is if none is set.

Reply:
tried expecting it to have `gap: 'normal normal'` but it didnt work. im going to remove the test for "does not has gap prop", but feel free to add it after merging the PR. also, pls check my comment on #30